util/src/main/java/com/liferay/faces/util/render/internal/RenderKitImpl.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-2015 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.util.render.internal;
+
+import javax.faces.render.RenderKit;
+import javax.faces.render.RenderKitWrapper;
+import javax.faces.render.Renderer;
+
+
+/**
+ * @author  Kyle Stiemann
+ */
+public class RenderKitImpl extends RenderKitWrapper {
```

Comment:
@ngriffin7a, perhaps this is overkill, but to make this code really rock solid, it would be better to have this `RenderKit` on the outside of the onion so that we ensure that the code in our `BodyRenderer` is always called. Would it be possible to manipulate the order of `RenderKit`s so that this one is on the outside? If we could do that with the BridgeImpl's `RenderKit` as well (so that the onion has the Bridge `RenderKit` as the outtermost and this second), then we would coexist with other component suites really well. It would also remove the need for [this code in the bridge](https://github.com/liferay/liferay-faces-bridge-impl/pull/5/files#diff-372a020be2eaf33a222e2c0ac5fc722bR204), which duplicates [the code above in the new Util `BodyRenderer`](https://github.com/liferay/liferay-faces-util/pull/6/files#diff-f5889f6a347668598344f9d6c87a5d70R52) because the Bridge's `BodyRenderer` could call through to this one but replace `<body>` tags with `<div>`s.